dietpi/dietpi-ddns

```diff
@@ -80,7 +80,7 @@ Read()
 		[[ $PASSWORD ]] || PASSWORD=${cronjob#*token=} PASSWORD=${PASSWORD%\'*}
 
 	# No-IP
-	elif [[ $cronjob == *'no-ip.com'* ]]
+	elif [[ $cronjob == *'noip.com'* ]]
```

Comment:
@ravenclaw900 
The API docs say "no-ip.com": https://www.noip.com/integrate/request
But indeed both work. They are pointing to the same IP, just slightly different TTL:
- https://dnschecker.org/all-dns-records-of-domain.php?query=dynupdate.no-ip.com&rtype=ALL&dns=cloudflare
- https://dnschecker.org/all-dns-records-of-domain.php?query=dynupdate.noip.com&rtype=ALL&dns=cloudflare